vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -3,54 +3,68 @@ sidebarDepth: 2
 ---
 
 # Create an Angular Micro Frontend
+This tutorial describes the process of building an Angular widget on Entando. It creates a docker-based bundle using the ent bundle CLI tool, which is the standard method starting with Entando 7.1.
 
-::: warning Prerequisites
-- [A working instance of Entando.](../../../docs/getting-started/README.md)
-:::
+Git-based bundles are also supported and to follow this tutorial in that format, see this [Angular tutorial](../../../../v7.0/tutorials/create/mfe/angular.md). 
 
-::: warning Tested Versions
-node v13.8.0 → We suggest using [nvm](https://github.com/nvm-sh/nvm) to handle node installations.
-:::
+## Prerequisites
+* [A working instance of Entando.](../../../docs/getting-started/README.md)
 
-## Create Angular App
+* node v13.8.0: Use [nvm](https://github.com/nvm-sh/nvm) to handle node installations.
 
-Install Angular CLI.
+## Initialize your Project Bundle
+1. To initialize your project bundle, give it any name you prefer and build the scaffolding
```

Comment:
"give it any name you prefer" is redundant without providing character or formatting constraints